opengrok-web/src/main/java/org/opengrok/web/api/v1/controller/ProjectsController.java

```diff
@@ -206,7 +206,7 @@ public void deleteProjectData(@PathParam("project") String projectName) throws H
         deleteHistoryCache(projectName);
 
         // Delete suggester data.
-        suggester.delete(projectName);
+        new Thread(() -> suggester.delete(projectName)).start();
```

Comment:
could you CompletableFuture.runAsync to possibly reuse resources

Reply:
done